src/viam/services/vision.py

```diff
@@ -134,52 +155,73 @@ async def get_detections(self, image: Union[Image, RawImage], detector_name: str
             confidence score of the labels, around the found objects in the next 2D image
             from the given camera, with the given detector applied to it.
         """
+        if extra is None:
+            extra = {}
         mime_type = CameraMimeType.JPEG
+        if isinstance(image, RawImage):
```

Comment:
@njooma these linting errors should've been found somehow with the earlier camera changes - is there anything we can do to make the linting more robust?

Reply:
We can have much stricter mypy policies as part of our test suite, which should catch these statically typed issues

Reply:
I'll add that to my todos

Reply:
thanks!